src/signatureUtils.js

```diff
@@ -0,0 +1,227 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const stateLib = require('@adobe/aio-lib-state')
+const fetch = require('node-fetch')
+const crypto = require('crypto')
+const loggerNamespace = '@adobe/aio-lib-events'
+const logger = require('@adobe/aio-lib-core-logging')(loggerNamespace,
+    { level: process.env.LOG_LEVEL })
+
+/**
+* Wrapper to check the event received by webhook
+* and return decoded (if encoded) and properly parsed payload
+* @param {*} event event payload received by webhook
+* @returns decoded and properly parsed payload
+*/
+function getProperPayload(event) {
```

Comment:
I am thinking why can't we move the functions `getProperPayload()`, `isBase64Encoded()` and `genErrorResponse()` to the more generic `helpers.js` file? Logically they can serve a bigger purpose than being limited to signature related methods.

Reply:
See https://dmitripavlutin.com/javascript-modules-best-practices/#31-the-problem-of-low-cohesion-modules.

Reply:
will do so. thanks for the article.

Reply:
Cool let me know once done, otherwise this PR look good overall.

Reply:
done